app/javascript/src/Metrics/components/ProductIndexPage.jsx

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import * as React from 'react'
+
+import { IndexPage } from 'Metrics'
+import { createReactWrapper } from 'utilities/createReactWrapper'
+import {
+  List,
+  ListItem
+} from '@patternfly/react-core'
+
+import type { Metric } from 'Types'
+
+type Props = {
+  applicationPlansPath: string,
+  createMetricPath: string,
+  mappingRulesPath: string,
+  metrics: Array<Metric>,
+  metricsCount: number
+}
+
+const ProductIndexPage = ({
+  applicationPlansPath,
+  createMetricPath,
+  mappingRulesPath,
+  metrics,
+  metricsCount
+}: Props): React.Node => {
+  const currentTabKey = (new URL(window.location.href)).searchParams.get('tab')
+  return (
+    <IndexPage
+      metrics={metrics}
+      metricsCount={metricsCount}
+      infoCard={currentTabKey === 'metrics' ? (
+        <>
+        Metrics track API usage. Metrics have these features:
+        <List>
+          <ListItem><i><b>Hits</b></i> is the 3scale native metric to which all methods report. This metric tracks the number of calls made to your API.</ListItem>
+          <ListItem>To track usage that does not increase the hit count, add top-level metrics.</ListItem>
+          <ListItem>To have specific calls to your API tracked by specific metrics, you must map a metric to one or more URL patterns listed in <a href={mappingRulesPath}>Mapping rules</a>.</ListItem>
+        </List>
+        </>
+      ) : (
+        <>
+          Methods track API usage.  Methods have these features:
+          <List>
+            <ListItem>Method calls trigger the <i>Hits</i> metric.</ListItem>
```

Comment:
```suggestion
            <ListItem>Method calls trigger the <b>Hits</b> metric.</ListItem>
```


Let's just stick to one font type, in this case bold, to indicate that Hits is a proper name in the context of 3scale.

Reply:
Regarding [1] keep in mind that although technically this is JS, once "compiled" it will be actual HTML.